### PR TITLE
added hover color on settings gear icon

### PR DESCRIPTION
**Pull request by AndersonCeci**
Added hover effect on gear icon on header
(Force Push was an accident)

### CODE REVIEW

File: src/app/header/header.component.css [hunk 1]
```diff
@@ -196,3 +201,17 @@
     font-size: 0.8rem;
     color: var(--dark-mode-smooth-theme);
 }
+
+.p-3 {
+    padding: 0.5rem !important;
```
Comment (AndiBraimllari) [inline src/app/header/header.component.css:206]: Is this a Bootstrap class? If so, add this code to settings-gear-button.

File: src/app/header/header.component.css [hunk 2]
```diff
@@ -152,9 +152,14 @@
 }
 
 .settings-gear-dark-mode {
+    margin-right: 2.3rem;
     color: var(--dark-mode-anti-theme);
 }
 
+.settings-gear-dark-mode:hover {
+    background-color: #545454 !important;
```
Comment (AndiBraimllari) [inline src/app/header/header.component.css:160]: Do we already have this color used somewhere? What does the off-canvas menu button use?

Comment (AndersonCeci) [reply src/app/header/header.component.css:160]: its the same color

Comment (AndiBraimllari) [reply src/app/header/header.component.css:160]: If it's fine with you, maybe we can create a variable for this?

Comment (AndersonCeci) [reply src/app/header/header.component.css:160]: sure ill go ahead and create one right now

**Pull request merged**